--- a/harmony/harmonydb/harmonydb.go
+++ b/harmony/harmonydb/harmonydb.go
@@ -25,6 +25,11 @@ import (
        "github.com/filecoin-project/curio/deps/config"
 )
 
+var logger = logging.Logger("harmonydb")
+
+//go:embed sql
+var fs embed.FS
+
 type ITestID string
 
 // ItestNewID see ITestWithID doc
@@ -41,8 +46,6 @@ type DB struct {
        BTFP      atomic.Uintptr // BeginTransactionFramePointer
 }
 
-var logger = logging.Logger("harmonydb")
-
 // NewFromConfig is a convenience function.
 // In usage:
 //
@@ -265,9 +268,6 @@ retry:
        return nil
 }
 
-//go:embed sql
-var fs embed.FS
-
 func (db *DB) upgrade() error {
        // Does the version table exist? if not, make it.
        // NOTE: This cannot change except via the next sql file.
		