--- a/pdp/contract/addresses.go	2025-07-03 12:06:40
+++ b/pdp/contract/addresses.go	2025-07-03 12:07:07
@@ -2,6 +2,7 @@
 
 import (
 	"math/big"
+	"os"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/snadrus/must"
@@ -16,6 +17,14 @@
 }
 
 func ContractAddresses() PDPContracts {
+
+	pdpAddr := os.Getenv("CURIO_PDP_VERIFIER_ADDRESS")
+	if pdpAddr != "" {
+		return PDPContracts{
+			PDPVerifier: common.HexToAddress(pdpAddr),
+		}
+	}
+
 	switch build.BuildType {
 	case build.BuildCalibnet:
 		return PDPContracts{
