--- a/pdp/contract/addresses.go
+++ b/pdp/contract/addresses.go
@@ -2,6 +2,7 @@ package contract
 
 import (
 	"math/big"
+	"os"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/snadrus/must"
@@ -32,6 +33,13 @@ func ContractAddresses() PDPContracts {
 				common.HexToAddress("0x81DFD9813aDd354f03704F31419b0c6268d46232"), // FilecoinWarmStorageService
 			},
 		}
+	case build.Build2k:
+		return PDPContracts{
+			PDPVerifier: common.HexToAddress(os.Getenv("PDP_VERIFIER_CONTRACT")),
+			AllowedPublicRecordKeepers: []common.Address{
+				common.HexToAddress(os.Getenv("RECORDKEEPER_CONTRACT")), // FilecoinWarmStorageService
+			},
+		}
 	default:
 		panic("PDP contract unknown for this network")
 	}
